mongodb/migrations/20210728-2.5.1.js

```diff
@@ -0,0 +1,41 @@
+var ObjectID = require("mongodb").ObjectID
+module.exports = {
+  async up(db, client) {
+    console.log(`migrate up`);
+
+    async function getCollection(collectionName) {
+      const result = []
+      await db.collection(collectionName).find({}).forEach(item => {
+        result.push(item)
+      })
+      return result
+    }
+
+    const lessons =  await getCollection("lessons");
+
+    lessons.forEach(lesson => {
+      if(lesson.image) {
+        lesson.media = {
+          link: lesson.image,
```

Comment:
let's not call this field `link` isn't `url` kind of the universal term? If you don't want `url` if can be `uri`